mingw.thread.h

```diff
@@ -30,22 +33,9 @@
 #include <process.h>
 #include <ostream>
 
-#ifdef _GLIBCXX_HAS_GTHREADS
-#error This version of MinGW seems to include a win32 port of pthreads, and probably    \

```

Comment:
I would like to preserve this as a warning

Reply:
See #33.